src/sdk-reference/cpp/1/websocket/intro/index.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: full.html.hbs
+title: Introduction
+order: 0
+---
+
+# Intro
+
+WebSocket is a class implementing the virtual class [Protocol]({{site_base_path}}sdk-reference/cpp/1/protocol).
+
+This class is used internally in the SDK and is not supposed to be used outside of it.
```

Comment:
It is meant to be used outside of this SDK internals, since it has to be instantiated+configured and then provided to the Kuzzle class constructor.

```suggestion
```

Also, why are the "internal" methods documented?